usr/share/lib/img_proof/tests/openSUSE_Leap/test_leap_license.py

```diff
@@ -1,5 +1,5 @@
 def test_leap_license(host):
-    license_dir = '/etc/YaST2/licenses/base'
+    license_dir = '/usr/share/licenses/openSUSE-release'
```

Comment:
This breaks older distros. Not sure we shouldn't make this conditional or look in both places.

Reply:
See #206 which fixes this.